docs/src/index.md

```diff
@@ -82,8 +82,44 @@ There are several core parts to the Catlab design, we start with a brief overvie
 
 4. [`@present`](@ref) enumerates a finite set of generators for a model of the GAT just like you would write out a group (model of the theory of groups) as list of generators and relations, the presentation lets you enumerate the objects and morphisms that generate a category.
 
+## Conventions
```

Comment:
I think we should definitely have a "for the developers" section of the docs, because most of the users of Catlab are also developers. Opened: https://github.com/AlgebraicJulia/Catlab.jl/issues/686#issue-1398437913